plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -528,6 +566,7 @@ private String createContainer(String workspaceId,
               .withEntrypoint(toArrayIfNotNull(service.getEntrypoint()))
               .withLabels(service.getLabels())
               .withNetworkingConfig(networkingConfig)
+              .withUser(cheUserId)
```

Comment:
Have you tried what will happen if you pass here empty string? Is it valid value for docker?